pages/login/[service].tsx

```diff
@@ -44,15 +44,14 @@ export default function AuthReturnPage({ services }) {
 
     if (!sent) {
       setSent(true)
-      login(dispatch, setError, router.query)
+      login(dispatch, toast.error, router.query)
```

Comment:
`login` still seems to be able to answer with untranslated strings?

Reply:
I am also not sure why the toast.error is passed here as a param, seems confusing

Reply:
ah just saw @razzeee comment on https://github.com/flathub/frontend/pull/260#discussion_r822748879